common/web/types/src/kvk/kvks-file-reader.ts

```diff
@@ -131,7 +131,7 @@ export default class KVKSFileReader {
           let key: VisualKeyboardKey = {
             flags: isUnicode ? VisualKeyboardKeyFlags.kvkkUnicode : 0, // TODO-LDML: bitmap support
             shift: shift,
-            text: sourceKey._ ?? '',
+            text: sourceKey.bitmap ? '' : (sourceKey._ ?? ''),
```

Comment:
Kind of curious what this is about.  To my knowledge, KMW doesn't have any knowledge of per-key "bitmaps" - be that image-bitmaps or bit masks (to choose a distinct word).

Reply:
Yes, that is right, KMW does not have support for per-key bitmaps. But .kvk and .kvks file formats do support per-key bitmaps, and these are used in Keyman for Windows.